src/agent/onefuzz/src/auth.rs

```diff
@@ -235,7 +235,6 @@ impl ManagedIdentityCredentials {
 #[derive(Clone, Debug, Deserialize)]
 struct ManagedIdentityAccessTokenBody {
     access_token: Secret<String>,
-    resource: String,
```

Comment:
Let's keep this with `allow`. We probably should log the resource name at a high verbosity level.